### DISCUSSION TITLE
Gateway api is accessible?

### DISCUSSION THREAD

**Discussion by zxyao145** [kubernetes-client/csharp#1196] [Q&A] [answered]
I want to try to use gateway api, but I don't konw where to find it.

**Comment by tg123** [comment, marked as answer]
gateway api is not included in official swagger, so you have to call with with crd
see [examples](https://github.com/kubernetes-client/csharp/tree/master/examples/customResource)

**Reply by zxyao145** [reply]
OK, thank you kindly!
